modules/nested-accordion/assets/scss/frontend.scss

```diff
@@ -4,15 +4,28 @@
 	--n-accordion-border-width: 1px;
 	--n-accordion-border-color: #{$gray-light};
 	--n-accordion-border-style: solid;
+  	--n-accordion-item-title-space-between: 0px;
+  	--n-accordion-item-title-distance-from-content: 0px;
+  	--n-accordion-padding-top:0px;
+  	--n-accordion-padding-right:0px;
+  	--n-accordion-padding-bottom:0px;
+  	--n-accordion-padding-left:0px;
 
 	.e-n-accordion {
 
 		&-item {
 
+		  padding: var(--n-accordion-padding-top) var(--n-accordion-padding-right) var(--n-accordion-padding-bottom) var(--n-accordion-padding-left);
+
+		  &:not(:last-child) {
+				margin-bottom: var(--n-accordion-item-title-space-between);
+		  	}
+
 			> .e-con {
 				border-width: var(--n-accordion-border-width);
 				border-color: var(--n-accordion-border-color);
 				border-style: var(--n-accordion-border-style);
+			  	margin-top: var(--n-accordion-item-title-distance-from-content);
```

Comment:
This might not be the best approach, as the .e-con element has margin settings that could overwrite this value. I think it might be safer to use gap styling on the wrapper.

Reply:
@davseve 
i need your input about naming css vars

Reply:
@davseve I wonder if we should use specific widget variables here or that it is better to use the container variables like `--border-width`. 
I know you have done more work on the relationship between nested widget container styling & container styling.

Possibly should we apply the low specificity aspect here as well?

Reply:
1.  Regarding the variable name, this is the exact same name we used in the mega menu, so I think that the var name is OK, unless we have a better suggestion.
2.  Regarding the border, we can use distributed CSS rules for each property, The problem is different, as Hein pointed out, we want the container's style to have a higher specificity than the accordion container style.
 We can't strengthen the specificity of the container, so we need to weaken accordion container specificity.
 In order to achieve that, we would like to assign the CSS styling in the selector in the PHP control.
 The selector should be defined with the `:where()` CSS function.

> The difference between :where() and [:is()](https://developer.mozilla.org/en-US/docs/Web/CSS/:is) is that :where() always has 0 [specificity](https://developer.mozilla.org/en-US/docs/Web/CSS/Specificity), whereas :is() takes on the specificity of the most specific selector in its arguments. ([https://developer.mozilla.org/en-US/docs/Web/CSS/:where](https://developer.mozilla.org/en-US/docs/Web/CSS/:where))

Like what we did in the nested carousel:
`$low_specificity_slider_container_selector = ':where( {{WRAPPER}} .swiper-slide ) > .e-con';
`

```
$this->add_group_control(
			Group_Control_Border::get_type(),
			[
				'name' => 'content_border',
				'selector' => $low_specificity_slider_container_selector,
				'fields_options' => [
					'color' => [
						'label' => esc_html__( 'Border Color', 'elementor-pro' ),
					],
					'width' => [
						'label' => esc_html__( 'Border Width', 'elementor-pro' ),
					],
				],
			]
		);`
`
```

Eventualy,